src/Compute/custom/NewAzVM_SimpleParameterSet.cs

```diff
@@ -0,0 +1,346 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Strategies;
+using Microsoft.Azure.Commands.Compute.Strategies;
+using Microsoft.Azure.Commands.Compute.Strategies.ComputeRp;
+using Microsoft.Azure.Commands.Compute.Strategies.Network;
+using Microsoft.Azure.Commands.Compute.Strategies.ResourceManager;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Network.Version2017_10_01;
+using Microsoft.Azure.Management.Internal.Resources;
+using Microsoft.Azure.Management.Internal.Resources.Models;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20190301;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Runtime;
+using Microsoft.Azure.Commands.Compute.Common;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Net;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System.Net.Http;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Strategies;
+
+namespace Microsoft.Azure.Commands.Compute
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VM_SimpleParameterSet", SupportsShouldProcess = true, DefaultParameterSetName = "SimpleParameterSet")]
+    [OutputType(typeof(VirtualMachine))]
+    [Microsoft.Azure.PowerShell.Cmdlets.Compute.Profile("latest-2019-04-30")]
+    public class NewAzVM_SimpleParameterSet: AzureRMAsyncCmdlet
+    {
+        public const string SimpleParameterSet = "SimpleParameterSet";
+
+        //[ResourceGroupCompleter]
+        [Parameter(
+            Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        //[LocationCompleter("Microsoft.Compute/virtualMachines")]
```

Comment:
@markcowl nit: same comment here -- remove this until we have the `LocationCompleter` implemented for generated cmdlets